### PR TITLE
Financial Connections: linted the files to avoid future merge conflicts.

### PR DESCRIPTION
**Pull request by kgaidis-stripe**
## Summary

^ linted the files to avoid future merge conflicts.

## Testing

Build & CI

I modified this:
 ` done < <(git diff --diff-filter=AM --name-only origin/master  | grep ".swift$")`

To this:
  `done < <(find . | grep "FinancialConnections" | grep ".swift$")`

And ran `ci_scripts/format_modified_files.sh`

### CODE REVIEW

File: StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift [hunk 1]
```diff
@@ -216,7 +224,9 @@ extension NativeFlowController {
                     switch result {
                     case .success(let session):
                         let eventType = "object"
-                        if session.accounts.data.count > 0 || session.paymentAccount != nil || session.bankAccountToken != nil {
+                        if !session.accounts.data.isEmpty || session.paymentAccount != nil
```
Comment (kgaidis-stripe) [inline StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift:227]: this is important to double check

File: StripeFinancialConnections/StripeFinancialConnections/Source/Native/Success/SuccessBodyView.swift [hunk 2]
```diff
@@ -30,7 +30,7 @@ final class SuccessBodyView: HitTestView {
         verticalStackView.axis = .vertical
         verticalStackView.spacing = 12
 
-        if linkedAccounts.count > 0 {
+        if !linkedAccounts.isEmpty {
```
Comment (kgaidis-stripe) [inline StripeFinancialConnections/StripeFinancialConnections/Source/Native/Success/SuccessBodyView.swift:33]: this is important to double-check

File: StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift (hunk 1 above)
Comment (vardges-stripe) [reply StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift:227]: Looks good, the only change is .count > 0 vs !isEmpty

**Review by vardges-stripe — APPROVED** [review]
MOST HELPFUL PR FOR FUTURE DEVELOPMENT

**Pull request merged**